tests/testthat/test-lint_package.R

```diff
@@ -16,17 +16,19 @@ test_that(
 
   # This dummy package does not have a .lintr file, so no files / lines should
   # be excluded from analysis
-  pkg_path <- file.path("dummy_packages", "assignmentLinter")
+  pkg_path <- test_path("dummy_packages", "assignmentLinter")
 
   expected_lines <- c(
     # from abc.R
     "abc = 123",
     # from jkl.R
-    "jkl = 456", "mno = 789"
+    "jkl = 456",
+    "mno = 789"
   )
   read_settings(NULL)
```

Comment:
This actually tests the default settings. (Related to the bug in the `.lintr` PR?)
Better to `skip_if(!is.null(find_config(...)))` and let `lint_package()` notice that the default config is needed?

Requires copying the package to a tempdir so our `.lintr` isn't found.

Reply:
sounds less trivial than this PR... follow-up? 😬

Reply:
Should be straight-forward now with the new solution of using `lintr_test_config`. That file is unlikely to exist in parent directories by accident.